tiled/client/base.py

```diff
@@ -370,6 +380,99 @@ def update_metadata(self, metadata=None, specs=None):
 
         This is subject to change or removal without notice
 
+        Parameters
+        ----------
+        metadata : dict, optional
+            User metadata. May be nested. Must contain only basic types
+            (e.g. numbers, strings, lists, dicts) that are JSON-serializable.
+            Hint:
```

Comment:
Consider replacing this explanation with an example:

```
Examples
--------

Make a mutable copy of the metadata, for editing.

>>> md = x.metadata_copy()

Add, remove, or change contents, e.g.

>>> md['a'] = 1
>>> del md['b']

Update the copy on the server.

>>> x.update_metadata(metadata)
```

Easier to quickly parse than an explanation.